x-pack/auditbeat/module/system/host/_meta/data.json

```diff
@@ -40,7 +40,18 @@
             },
             "timezone.name": "UTC",
             "timezone.offset.sec": 0,
-            "uptime": 222611977726
+            "uptime": 222611977726,
+            "users": [
```

Comment:
I'm worried about adding nested objects to our data structure. What we normally do in these cases send separate events for each entry.

Reply:
This would be another place where not using a nested document would allow us to map some of the fields to [ECS](https://github.com/elastic/ecs#user) (like user.id and user.name).